app/views/ChargeSummary.scala.html

```diff
@@ -273,7 +273,7 @@ <h1 class="govuk-heading-xl">
         <div id="payment-link-@{taxYearTo}">
             <a class="govuk-button" role="button" data-module="govuk-button" href="@controllers.routes.PaymentController.paymentHandoff(documentDetail.remainingToPay.toPence, origin)">
                 @messages("paymentDue.payNow")
-                <span class="govuk-visually-hidden">@messages("paymentDue.pay-now-hidden", taxYearFrom, taxYearTo)</span>
+                <span class="govuk-visually-hidden">@messages("paymentDue.pay-now-hidden", taxYearFrom.toString, taxYearTo.toString)</span>
```

Comment:
can't find this in frontend + is of class="govuk-visually-hidden" so is maybe hidden anyway. tests pass so should I keep the change or just remove it as it doesn't really matter?

Reply:
Double check relevant documentation for more details
https://frontend.design-system.service.gov.uk/sass-api-reference/#example-using-a-custom-ratio